std/format.d

```diff
@@ -6321,7 +6321,9 @@ private size_t guessLength(Char, S)(S fmtString)
 
         if (spec.width == spec.precision)
             len += spec.width;
-        else if (spec.width > 0 && (spec.precision == spec.UNSPECIFIED || spec.width > spec.precision))
+        else if (spec.width > 0 && spec.width != spec.DYNAMIC &&
+            (spec.precision == spec.UNSPECIFIED || spec.width > spec.precision)
+        )
```

Comment:
Minor style nitpick: I'm pretty sure this violates Phobos style (though I could be wrong). I'd just put the closing parens at the end of the previous line, and use braces around the next line.

Either that, or indent the opening parens of this line with the first `spec` of the previous line, so that it's clear that it doesn't belong with the next line.

Reply:
Better like this?